modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/js/main.js

```diff
@@ -811,91 +811,69 @@ AUI.add(
 					var instance = this;
 
 					var cmdNode = instance.byId('cmd');
-					var portletURL;
 					var redirectNode = instance.byId('redirect');
 
 					if (cmdNode.val() === 'add' || cmdNode.val() === 'update') {
-						portletURL = Liferay.PortletURL.createURL(
-							redirectNode.val()
-						);
+						var params = {
+							cmd: cmdNode.val()
+						};
 
-						portletURL.setParameter('cmd', cmdNode.val());
+						var url = Liferay.Util.PortletURL.createPortletURL(
+							redirectNode.val(),
+							params
+						);
 
 						if (instance._exportLAR) {
-							portletURL.setParameter(
-								'mvcRenderCommandName',
-								'editExportConfiguration'
-							);
-							portletURL.setParameter(
-								'tabs2',
-								'new-export-process'
-							);
-							portletURL.setParameter(
-								'exportConfigurationButtons',
-								'custom'
-							);
+							params.mvcRenderCommandName =
+								'editExportConfiguration';
+							params.tabs2 = 'new-export-process';
+							params.exportConfigurationButtons = 'custom';
 						} else {
-							portletURL.setParameter(
-								'mvcRenderCommandName',
-								'editPublishConfiguration'
-							);
-							portletURL.setParameter(
-								'publishConfigurationButtons',
-								'custom'
-							);
+							params.mvcRenderCommandName =
+								'editPublishConfiguration';
+							params.publishConfigurationButtons = 'custom';
 						}
 
 						var groupIdNode = instance.byId('groupId');
 
 						if (groupIdNode) {
-							portletURL.setParameter(
-								'groupId',
-								groupIdNode.val()
-							);
+							params.groupId = groupIdNode.val();
 						}
 
 						var liveGroupIdNode = instance.byId('liveGroupId');
 
 						if (liveGroupIdNode) {
-							portletURL.setParameter(
-								'liveGroupId',
-								liveGroupIdNode.val()
-							);
+							params.liveGroupId = liveGroupIdNode.val();
 						}
 
 						var privateLayoutNode = instance.byId('privateLayout');
 
 						if (privateLayoutNode) {
-							portletURL.setParameter(
-								'privateLayout',
-								privateLayoutNode.val()
-							);
+							params.privateLayout = privateLayoutNode.val();
 						}
 
 						var rootNodeNameNode = instance.byId('rootNodeName');
 
 						if (rootNodeNameNode) {
-							portletURL.setParameter(
-								'rootNodeName',
-								rootNodeNameNode.val()
-							);
+							params.rootNodeName = rootNodeNameNode.val();
 						}
 
-						redirectNode.val(portletURL.toString());
+						redirectNode.val(url.toString());
 					}
 
 					if (cmdNode) {
 						var form = instance.get('form');
 
-						portletURL = Liferay.PortletURL.createURL(
-							form.get('action')
+						url = Liferay.Util.PortletURL.createPortletURL(
+							form.get('action'),
+							{
+								p_p_lifecycle: 0
```

Comment:
Instead of setting `p_p_lifecycle`, we should use `createRenderURL`, see https://github.com/liferay/liferay-portal/blob/a20141d838cb8897a68d62fe2154467332baea69/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/portlet_url/create_render_url.es.js#L24